src/components/FormBuilder/FormBuilder.tsx

```diff
@@ -9,31 +9,39 @@ import PasswordInput from "../PasswordInput/PasswordInput";
 import ErrorBlock from "../ErrorBlock/ErrorBlock";
 import { useInput } from "src/hooks/useInput";
 
+import "./style.scss";
+
 interface FormBuilderProps {
   config: ConfigProps[];
   setInputValues?: Function;
   aboveLink?: React.ReactNode;
-  label: string;
+  label?: string;
+  incorrect?: React.ReactNode;
```

Comment:
Please rename it to "incorrectLabel" or smth like that

Reply:
Done.